thingsToImplement/redux/reduxStore.js

```diff
@@ -1,6 +1,38 @@
 
-import {createStore} from 'redux';
+import {createStore, applyMiddleware} from 'redux';
 import reducers from './combinedReducers.js';
+ 
 
-const store = createStore(reducers);
+// const {setPort, setItemsToSend, setServerOn} = require('../../public/xmainWindow.js');
+import {setURLFilePath} from '../../src/main/xserver/xroutes/xroutes.js';
+import * as types from './actionTypes.js';
+
+
+// const savePort = store => next => action => {
```

Comment:
Again, no need for commented code, bruh.

Reply:
I left that commented code in there because we may potentially need it at some point, and I wanted to make sure that I wouldn't completely forget about its existence. I also figured that since this is being merged with a dev branch, the commented code was ok to leave in there.

Reply:
🤷‍♂ I'll remove it later.